lib/i18n/strings.g.dart

```diff
@@ -521,6 +523,19 @@ class _StringsSettingsLayoutSizesEn {
 	String get tablet => 'Tablet';
 }
 
+// Path: settings.layoutSizes
+class _StringsSettingsEditorToolbarSizesEn {
```

Comment:
I think you forgot to commit `strings.i18n.yaml`

Reply:
added strings.i18n.yaml (added manually string for toolbar button size).  Please correct it if i did it wrong.